src/cargo/ops/cargo_compile/mod.rs

```diff
@@ -556,6 +556,21 @@ where `<compatible-ver>` is the latest version supporting rustc {rustc_version}"
         }
     }
 
+    if build_config.detect_antivirus {
+        // TODO(madsmtm): Maybe only do this when we have above a certain
+        // number of build scripts or test binaries to run?
+
+        // TODO(madsmtm): We probably don't want to do this check when doing
+        // `cargo install`?
+
+        // TODO(madsmtm): Consider only warning once every X days.
```

Comment:
Calling these out to make sure they get tracked

Reply:
There's a few more, I'll add all of it to the tracking issue once we create one (which I'm not sure I should yet?)